src/store/plugins/load-geojson-style-and-data.plugin.js

```diff
@@ -54,20 +92,33 @@ async function loadDataAndStyle(geoJsonLayer) {
  */
 export default function loadGeojsonStyleAndData(store) {
     store.subscribe((mutation) => {
-        const addLayersSubscriber = async (layers) => {
-            const updatedLayers = await layers
-                .filter((layer) => layer instanceof GeoAdminGeoJsonLayer && layer.isLoading)
-                .map((layer) => {
-                    log.debug(`Loading data/style for added GeoJSON layer`, layer)
-                    return loadDataAndStyle(store, layer)
+        const addLayersSubscriber = (layers) => {
+            layers
+                .filter((layer) => layer instanceof GeoAdminGeoJsonLayer)
+                .forEach((layer) => {
+                    if (layer.isLoading) {
+                        log.debug(`Loading data/style for added GeoJSON layer`, layer)
+                        loadDataAndStyle(store, layer)
+                    }
+                    if (layer.updateDelay > 0) {
+                        log.debug('starting auto-reload of data for layer', layer)
+                        autoReloadData(store, layer)
+                    }
                 })
-            store.dispatch('updateLayers', { layers: updatedLayers, ...dispatcher })
```

Comment:
The single `updateLayers` dispatch should be kept for performance reason during startup.